### ISSUE TITLE
[feature] CHA-Based Virtual Call Navigation

### ISSUE THREAD

**Issue by konata** [skylot/jadx#2311]
## Description
Implement a feature for navigating virtual call sites using Class Hierarchy Analysis (CHA) for the following sources:
```kotlin
abstract class Foo {
  abstract fun foo(): String
}

class Foo1 : Foo() {
  override fun foo(): String {
    TODO("Not yet implemented")
  }
}

class Foo2 : Foo() {
  override fun foo(): String {
    TODO("Not yet implemented")
  }
}

fun foobar(foo: Foo) {
    foo.foo() // call site
}
```

## Proposed Behavior
When navigating to the call site `foo.foo()`, instead of a direct jump to `Foo.foo()`, display a selection menu that includes:

- Foo1.foo()
- Foo2.foo()
- Foo.foo()

This allows users to choose which implementation they want to navigate to, and the same functionality should apply to interface methods as well.

**Comment by MrIkso** [comment]
@skylot hi, i think this issues not releated to this project

**Comment by skylot** [comment]
@MrIkso it is good request and jadx already collect all needed data (used in override analysis).
Although, I think implementation will be slightly different. Using common 'references' window (#1368) will be easier to implement. So instead of jump menu, that 'references' window needs to be opened.

In short: this issue will be implemented as a part of #1368.

**Comment by MrIkso** [comment]
@skylot ah, it's related to xrefs, I just didn't understand this issue. Sorry.
